contents/docs/product-analytics/group-analytics.mdx

```diff
@@ -24,10 +24,118 @@ import JSSetGroupProperties from "../\_snippets/setting-group-properties.mdx"
 
 <JSCreateGroups />
 
-### Setting and updating group properties
+## How to set group properties
 
 <JSSetGroupProperties />
 
+## How to capture group events
+
+Events are automatically linked to the groups that the user who submitted the event belongs to. Thus you do not add any additional code once you have linked a user to the group. This is the recommended way to capturing group events.
+
+To link a user to a group, call the same code as you would when [creating a group](#how-to-create-groups).
+
+### Advanced (server-side only): Capturing group events without a user
+
+If you want to capture group events but don't want to associate them with a specific user, there are two alternatives:
+
+#### Method one (recommended): Use the group ID as the event's distinct ID
```

Comment:
Why is this recommended?

I think last I discussed with @tiina303 , our recommendation was to create a proxy distinct ID, like "group_events" that is used for all group calls without a user, so we don't generate random useless people.

The confusion here might be that our SDKs don't follow this best practice, because earlier indeed we were doing the same 🙈

Reply:
Oh wait, I think I get it now: You can assign the event to a group, and it doesn't matter if the `distinct_id` of the user who submitted the event is assigning different groups to different events (I was under the impression that 1 user = 1 group. But I get it now that users and groups are not linked)

Reply:
Exactly